recipes/m2crypto/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: m2crypto
+  version: !!str 0.23.0
+
+source:
+  fn: M2Crypto-0.23.0.tar.gz
+  url: https://pypi.python.org/packages/source/M/M2Crypto/M2Crypto-0.23.0.tar.gz
```

Comment:
Using jinja for versions would be a good change that should help keep this maintainable. Here's an [example]( https://github.com/conda-forge/python-coveralls-feedstock/blob/19ac650e9e47db66cdb27313b5591beab247c5e6/recipe/meta.yaml#L1-L9 ).